src/app/stylesheets/theme.less

```diff
@@ -12,40 +12,40 @@
 /*                       Twitter Bootstrap colors                        */
 /*-----------------------------------------------------------------------*/
 @brand-success:                  #33d375;
-@brand-warning:                  @mno-demo-orange;
-@brand-info:                     @mno-demo-aqua-green;
-@brand-danger:                   @mno-demo-strawberry;
-@brand-primary:                  @mno-demo-aqua-green;
+@brand-warning:                  #F2B469;
```

Comment:
Yes but it is not compatible with the color pickers.

Using variables:
![screen shot 2017-06-21 at 4 58 18 pm](https://user-images.githubusercontent.com/3941996/27521864-9e6fcd90-5a64-11e7-87fc-48adb84844ef.png)

Using color hexa:
![screen shot 2017-06-22 at 5 44 36 pm](https://user-images.githubusercontent.com/3941996/27521879-b50849c4-5a64-11e7-928f-41018efd1eb5.png)